Python/optimizer.c

```diff
@@ -871,22 +889,25 @@ uop_optimize(
     _PyBloomFilter dependencies;
     _Py_BloomFilter_Init(&dependencies);
     _PyUOpInstruction trace[_Py_UOP_MAX_TRACE_LENGTH];
-    int trace_length = translate_bytecode_to_trace(code, instr, trace, _Py_UOP_MAX_TRACE_LENGTH, &dependencies);
-    if (trace_length <= 0) {
+    int err = translate_bytecode_to_trace(code, instr, trace, _Py_UOP_MAX_TRACE_LENGTH, &dependencies);
+    if (err <= 0) {
         // Error or nothing translated
-        return trace_length;
+        return err;
     }
     OPT_HIST(trace_length, trace_length_hist);
```

Comment:
If `Py_STATS` is defined, this line still uses `trace_length`, but that variable is undefined. We probably shouldn't have this statistic, *or* move it into `translate_bytecode_to_trace()`.